controllers/sr_controller/sr_controller.py

```diff
@@ -43,10 +51,14 @@ def get_robot_file(zone_id: int, mode: str) -> Path:
           are found it copies an example into place (at the root) and uses that.
     """
 
-    robot_file = controller_utils.get_zone_robot_file_path(zone_id)
-    fallback_robot_file = controller_utils.ARENA_ROOT / "robot.py"
+    robot_file = controller_utils.get_zone_robot_file_path(zone_id, robot_type)
     strict_zones = STRICT_ZONES[mode]
 
+    if robot_type == RobotTypes.CRANE:
+        fallback_robot_file = controller_utils.ARENA_ROOT / "crane.py"
```

Comment:
Nit: Could just use `robot_type.value` here, too.

Reply:
Fixed in b1b0844b88e8a1103d4180372d9c5d49ee023c32